TESTING.md

```diff
@@ -118,3 +118,30 @@ The "Message Evaluation Procedure" is:
     1. Under "Configured templates", find your template and click the wrench icon.
     2. Enter a new template name.
     3. _Observe_: The name updates.
+
+## Visual Regression tesing
```

Comment:
also add "with BackstopJS" to the title, and add a bit of context before starting with step-by-step instructions (this extension uses BackstopJS (link) to check etcetc), otherwise people might not know what you are talking about